### ISSUE TITLE
"ATLauncher.app" cannot be opened because the developer cannot be verified

### ISSUE THREAD

**Issue by The-Fireplace** [ATLauncher/ATLauncher#501]
**Describe the bug**
On macOS, the ATLauncher.app file cannot be easily opened by default.

**To Reproduce**

1. Download the OS X Application or the Jar from the ATLauncher website
2. Run it.
3. See that there is an error

**Expected behavior**
The app launches without this error

**Screenshots**
![Screen Shot 2021-08-18 at 1 01 06 PM](https://user-images.githubusercontent.com/5914284/129949237-d1574850-1d4a-43a3-8666-3f229dad6142.png)
![Screen Shot 2021-08-18 at 1 07 49 PM](https://user-images.githubusercontent.com/5914284/129949916-d6519ee5-9e97-405e-840f-24aed0551e39.png)

**System information:**
- OS: Mac OSX Big Sur

**Additional context**
Workaround: 
1. Attempt to run the file
2. Open System Settings
3. Click Security & Privacy
4. Next to the message that says '"ATLauncher" was blocked from use because it is not from an identified developer', press Open Anyway.
5. Press Open

**Comment by RyanTheAllmighty** [comment]
Yup this is apples pay wall. Since we don't pay for their developer program to sign the application, this is the unfortunate side effect.

One day we might sign, but it's not in the plan for now

**Comment by The-Fireplace** [comment]
Alright, no problem. I figured getting it signed so they recognize it might be behind a paywall, good to know for sure.

**Comment by RyanTheAllmighty** [comment]
Yeah overall I'm not too happy with the OSX app, but without having the ability to test/dev on the hardware, I can't easily fix up the issues I have with the app enough to be happy enough to start signing it. But yeah definitely in the plans to eventually support
